jwala-integration-test/src/test/resources/com/cerner/jwala/ui/selenium/configuration/search.feature

```diff
@@ -0,0 +1,122 @@
+Feature: Search
+
+  Scenario: Search within groups
+    Given I logged in
+    And I am in the configuration tab
+    And I am in the group tab
+    And I created a group with the name "MMMGroup"
+    And I created a group with the name "ZZZGroup"
+    When I fill in the search field with "MM"
+    Then I see "MMMGroup" in the group table
+    And I don't see "ZZZGroup" in the table
+
+
+  Scenario: Search within web app
+    Given I logged in
+    And I am in the configuration tab
+    And I am in the group tab
+    And I created a group with the name "group1"
+    And I created a group with the name "group2"
+    And I am in the web apps tab
+    And I created a web app with the following parameters:
+      | webappName  | application |
+      | contextPath | \name1       |
```

Comment:
use '/' and make the contextPath more descriptive e.g. "aContextPath1"  e.g.

    | contextPath | /name1 |